doc/source/whatsnew/v0.18.2.txt

```diff
@@ -527,3 +527,5 @@ Bug Fixes
 
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
+
```

Comment:
put your whatsnew higher up (in an empty space in Bug Fixes) this way you dont have conflicts